docs/en/source/_build/html/_sources/onepageTutorial.rst.txt

```diff
@@ -0,0 +1,327 @@
+
+*****************************************
+ZooKeeper Installation
+*****************************************
+
+
+1. Go to your ZooKeeper config directory. For me its C:\zookeeper-3.4.7\conf
+2. Rename file “zoo_sample.cfg” to “zoo.cfg”
+3. Open zoo.cfg in any text editor, like Notepad; I prefer Notepad++.
+4. Find and edit dataDir=/tmp/zookeeper to :\zookeeper-3.4.7\data  
+5. Add an entry in the System Environment Variables as we did for Java.
+
+ a. Add ZOOKEEPER_HOME = C:\zookeeper-3.4.7 to the System Variables.
+ b. Edit the System Variable named “Path” and add ;%ZOOKEEPER_HOME%\bin; 
+
+6. You can change the default Zookeeper port in zoo.cfg file (Default port 2181).
+7. Run ZooKeeper by opening a new cmd and type zkserver.
+8. You will see the command prompt with some details, like the image below:
+
+.. figure:: figures/.png
+
+
+*****************************************
+Setting Up Kafka
+*****************************************
+
+1. Go to your Kafka config directory. For me its **C:\kafka_2.11-0.9.0.0\config**
+2. Edit the file “server.properties.”
+3. Find and edit the line log.dirs=/tmp/kafka-logs” to “log.dir= C:\kafka_2.11-0.9.0.0\kafka-logs.
+4. If your ZooKeeper is running on some other machine or cluster you can edit “zookeeper.connect:2181” to your custom IP and port. For this demo, we are using the same machine so there's no need to change. Also the Kafka port and broker.id are configurable in this file. Leave other settings as is.
+5. Your Kafka will run on default port 9092 and connect to ZooKeeper’s default port, 2181.
+
+
+*****************************************
+Running a Kafka Server
+*****************************************
+
+Important: Please ensure that your ZooKeeper instance is up and running before starting a Kafka server.
+
+1. Go to your Kafka installation directory:** C:\kafka_2.11-0.9.0.0\**
+2. Open a command prompt here by pressing Shift + right click and choose the “Open command window here” option).
+3. Now type **.\bin\windows\kafka-server-start.bat .\config\server.properties** and press Enter.
+
+ **.\bin\windows\kafka-server-start.bat .\config\server.properties**
+
+*****************************************
+Setting up PostgreSQL
+*****************************************
+
+Step 1) Go to https://www.postgresql.org/download and select O.S. ,it's Windows for me.
+
+
+Step 2) You are given two options:-
+
+ 1. Interactive Installer by EnterpriseDB
+ 2. Graphical Installer by BigSQL
+
+BigSQL currently installs pgAdmin version 3 which is deprecated. It's best to choose EnterpriseDB which installs the latest version 4
+
+
+Step 3)
+
+ 1.You will be prompted to desired Postgre version and operating system. Select the latest 
+ Postgre version and OS as per your environment
```

Comment:
We should specify a known working version. From the top level readme:
Scorpio has been tested and developed with Postgres 10.

Reply:
version details added.